core/src/test/java/io/undertow/server/handlers/accesslog/AccessLogFileTestCase.java

```diff
@@ -213,4 +213,30 @@ public void testForcedLogRotation() throws IOException, InterruptedException {
         }
     }
 
+    @Test
+    public void testLoggingDuplicateHeaders() throws IOException, InterruptedException {
+        Path logFileName = logDirectory.resolve("server1.log");
```

Comment:
The test failures are because windows may not have released the lock on the file from previous tests, each test needs to use a new file name.